honeybadgermpc/ntl/helpers.h

```diff
@@ -208,4 +208,144 @@ void fnt_decode_step2(vec_ZZ_p &P_coeffs, ZZ_pX &A, vec_ZZ_p &Ad_evals,
     for (int i=0; i < k; i++) {
         P_coeffs[i] = coeff(P, i);
     }
```

Comment:
If anything... `helpers.h` has a too generic name. It's actually chock full of some interesting algorithms of independent interest implemented using NTL. What if we rename it as `rsdecoder_impl.h` or something like that, just so it is a bit more descriptive what's in there?